css/ndt-test.css

```diff
@@ -0,0 +1,366 @@
+@font-face {
+  font-family: "League Gothic";
+  src: url("../fonts/league-gothic/league_gothic.otf") format("opentype");
+}
+
+body * {
+  margin: 0;
+  padding: 0;
+}
+
+body {
```

Comment:
My understanding is that they inherit. Can we delete 6-9 and see if you get the same behavior? If so, leave deleted, otherwise keep.